algorithm/kapacity/timeseries/forecasting/train.py

```diff
@@ -0,0 +1,130 @@
+#  Copyright 2023 The Kapacity Authors.
+#
+#  Licensed under the Apache License, Version 2.0 (the "License");
+#  you may not use this file except in compliance with the License.
+#  You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS,
+#  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#  See the License for the specific language governing permissions and
+#  limitations under the License.
+
+
+import argparse
+import yaml
+import datetime
+import pandas as pd
+from google.protobuf import timestamp_pb2
+
+import kapacity.portrait.horizontal.predictive.main as pred_main
+import kapacity.timeseries.forecasting.forecaster as forecaster
+
+
+def main():
+    # 1. parse arguments
+    args = parse_args()
+
+    # 2. fetch train config
+    config = load_config(args.train_config_file)
+
+    # 3. fetch training data set
+    df = fetch_train_data(args, config)
+
+    # 4. train the model using data sets
+    train(args, config, df)
+
+    return
+
+
+def parse_args():
+    parser = argparse.ArgumentParser(description='args of timeseries forcasting training')
+    parser.add_argument('--train-data-file', help='training data set file', required=False)
```

Comment:
```suggestion
    parser.add_argument('--dataset-file', help='path of training dataset file, if set, will load dataset from this file instead of fetching from metrics server', required=False)
```